File: C++/Basic/Numbers/GreatestSmallest.cpp
/*Checking the greatest number using C++*/

#include <iostream>
using namespace std;\
int main()
{
    int a,b;
    cout << "Enter the value of a and b respectively \n";
    cin >> a >> b;
    if(a>b)
    cout << "The greatest number is 'a' whose value is " << a << " While the smallest number is 'b' whose value is " << b;
    else
    cout << "The greatest number is 'b' whose value is " << b << " While the smallest number is 'a' whose value is " << a;
    return 0;
}

File: Java/basics/Numbers/GreatestSmallest.java
/*Checking the greatest and smallest number using C++*/

import java.util.*;
public class GreatestSmallest
{
    public static void main (String args[])
    {
        Scanner in = new Scanner(System.in);
        int a,b;
        System.out.println("Enter the value of a and b respectively");
        a=in.nextInt();
        b=in.nextInt();
        if(a>b)
        {
            System.out.println("The greatest number is a whose value is "+a);
            System.out.println("The smallest number is b whose value is "+b);
        }
        else
        {
            System.out.println("The greatest number is b whose value is "+b);
            System.out.println("The smallest number is a whose value is "+a);
        }
    }
}
